lmp/bb-config.sh

```diff
@@ -36,6 +37,12 @@ fi
 
 source setup-environment build
 
+if [ "$DEV_MODE" == "1" ]; then
+	cat << EOFEOF >> conf/local.conf
+DEV_MODE="1"
```

Comment:
Add space between = to be similar to the over local.conf variables.

Reply:
@ricardosalveti done!